earth_enterprise/src/server/wsgi/common/utils.py

```diff
@@ -23,7 +23,7 @@
 import re
 import socket
 import urllib2
-import xml.sax.saxutils as saxutils
+import defusedxml.sax.saxutils as saxutils
```

Comment:
The server is choking on this line. Apparently defusedxml doesn't implement saxutils. It might be best to create some kind of "wrapper package" that pulls the parsing functions from defusedxml and the other functions from other libraries. I haven't tested this at all but I think a file similar to the below would work:
```
import defusedxml
from xml.sax import saxutils

defusedxml.sax.saxutils = saxutils
# Other substitutions as needed
```
Then we could import the above file instead of defusedxml and use it like defusedxml.

Reply:
I guess in this particular case we can just go back to the previous import. A wrapper might be useful if we have to make several replacements, though.

Reply:
There's an example of this here: https://wsgidav.readthedocs.io/en/latest/_modules/wsgidav/xml_tools.html. Basically, we could put something like the following code in a file and import it instead of defusedxml.
```
    from defusedxml import ElementTree as etree

    # defusedxml doesn't define these non-parsing related objects
    from xml.etree.ElementTree import Element, SubElement, tostring

    etree.Element = _ElementType = Element
    etree.SubElement = SubElement
    etree.tostring = tostring
```
I haven't tried this at all - it's just an idea at this point. I have no idea what the problems or pitfalls might be.